drupal/rootfs/etc/nginx/shared/drupal.defaults.conf

```diff
@@ -85,6 +85,11 @@ location ~ ^(/[a-z\-]+)?/system/files/ { # For Drupal >= 7
     try_files $uri /index.php?$query_string;
 }
 
+# handle CSS/JS aggregation through Drupal
+location ~ ^/sites/default/files/(css|js)/ {
```

Comment:
Can you replace "default" with ".*" so that aggregation can work with multi-sites.

Reply:
Or it could be merged with line 78, which was the previous support for styles generation.

Reply:
Good call 👍 https://github.com/Islandora-Devops/isle-buildkit/pull/337/commits/a6916e6752ad152fd7031a5f23da0d7e7ca8df5a